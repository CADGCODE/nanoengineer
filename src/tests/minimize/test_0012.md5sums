stdout 5096A1539118611AE8EDCA7ECCC29F3C
stderr D41D8CD98F00B204E9800998ECF8427E
test_0012.xyz 577B399F220B9B79B0B7C5AC00E1AF62
