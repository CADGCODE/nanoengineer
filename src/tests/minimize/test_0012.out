======= test_0012.test =======
======= exitvalue =======
0
======= stderr =======
======= stdout =======
iters per frame = 10
number of frames = 100
timestep = 1.000000e-16
temp = 0.000000
dump as text
< test_0012.mmp  > test_0012.xyz

Total Ke = 9.315000e-20
final RMS gradient=0.985895 after 96 iterations
======= test_0012.trc =======
# nanoENGINEER-1.com Simulator Trace File, Version 050310
#
# Input File:test_0012.mmp
# Output File: test_0012.xyz
# Trace File: test_0012.trc
# Number of Atoms: 15
# Number of Frames: 100
# Steps per Frame: 10
# Temperature: 300.0
# 
# 0 columns:
#

# rms force, high force
1 1675.26 2571.42
2 1197.40 1819.05
3 735.22 1186.77
4 577.26 926.85
5 521.18 879.48
6 408.21 684.04
7 363.03 652.03
8 320.31 559.23
9 278.01 501.24
10 240.12 399.75
11 238.25 366.72
12 215.63 349.14
13 156.14 275.91
14 142.39 250.70
15 128.67 226.79
16 114.06 200.01
17 98.64 172.26
18 84.21 143.48
19 82.22 120.12
20 80.15 138.19
21 56.14 93.39
22 50.21 86.50
23 45.13 76.71
# Switching to Conjugate-Gradient
24 15.10 0.00
25 6.54 0.00
26 6.49 0.00
27 6.25 0.00
28 3.85 0.00
29 2.92 0.00
30 2.03 0.00
31 1.92 0.00
32 1.91 0.00
33 1.88 0.00
34 1.86 0.00
35 1.85 0.00
36 1.84 0.00
37 1.83 0.00
38 1.83 0.00
39 1.82 0.00
40 1.82 0.00
41 1.82 0.00
42 1.82 0.00
43 1.82 0.00
44 1.82 0.00
45 1.82 0.00
46 1.81 0.00
47 1.81 0.00
48 1.81 0.00
49 1.81 0.00
50 1.81 0.00
51 1.81 0.00
52 1.80 0.00
53 1.80 0.00
54 1.79 0.00
55 1.78 0.00
56 1.77 0.00
57 1.77 0.00
58 1.77 0.00
59 1.77 0.00
60 1.76 0.00
61 1.76 0.00
62 1.74 0.00
63 1.72 0.00
64 1.72 0.00
65 1.72 0.00
66 1.71 0.00
67 1.71 0.00
68 1.71 0.00
69 1.71 0.00
70 1.71 0.00
71 1.70 0.00
72 1.69 0.00
73 1.68 0.00
74 1.67 0.00
75 1.66 0.00
76 1.61 0.00
77 1.55 0.00
78 1.51 0.00
79 1.46 0.00
80 1.44 0.00
81 1.42 0.00
82 1.41 0.00
83 1.40 0.00
84 1.39 0.00
85 1.36 0.00
86 1.29 0.00
87 1.23 0.00
88 1.16 0.00
89 1.11 0.00
90 1.08 0.00
91 1.06 0.00
92 1.04 0.00
93 1.02 0.00
94 1.00 0.00
95 0.99 0.00
96 0.99 nan
# Done: Minimization final rms: 0.985895, highForce: nan
======= test_0012.xyz =======
15
RMS=0.985895
C 0.040107 0.023129 -0.013012
X -0.429606 0.478858 -0.453425
X 0.493496 -0.442647 -0.459991
C 0.947002 0.925118 0.876707
X 0.772237 1.681366 0.740613
X 1.702757 0.768676 0.717592
C -0.832876 -0.858196 0.929499
X -0.676596 -1.618066 0.791113
X -1.593454 -0.691457 0.808257
C 0.650751 0.585626 2.367839
X 0.388819 1.232734 2.733392
X 1.306344 0.306995 2.704820
C -0.454599 -0.511050 2.399981
X -1.088861 -0.229472 2.771587
X -0.171523 -1.154615 2.754028
